packages/integration/src/gitlab/core.test.ts

```diff
@@ -186,4 +186,48 @@
       });
     });
   });
+
+  describe('getGitLabRequestOptions', () => {
+    it('should return Authorization header when oauthToken is provided', () => {
+      const token =
+        'de6780bc506a0446309bd9362820ba8aed28aa506c71eedbe1c5c4f9dd350e54';
```

Comment:
Not really a valid credential as it can be seen here: https://docs.gitlab.com/ee/api/oauth2.html#authorization-code-with-proof-key-for-code-exchange-pkce